torch/fx/passes/splitter_base.py

```diff
@@ -1092,6 +1104,8 @@ def split(self, remove_tag: bool = False) -> torch.fx.GraphModule:
     def __call__(self) -> torch.fx.GraphModule:
         subgraphs = self.put_nodes_into_subgraphs()
         subgraphs = self.remove_small_acc_subgraphs(subgraphs)
+        if self.settings.move_non_tensor_nodes_on_boundary:
+            move_non_tensor_nodes_on_boundary(subgraphs)
```

Comment:
i'm wondering if it's possible to integrate this with the node tracker in some way,

as here we're changing the split result, and the node movements we tracked in it will actually be wrong

```
        self.tracker = NodeEventTracker(int(TRACKER_MODE), DUMP_PREFIX)
```